keyboards/planck/keymaps/oryx/rules.mk

```
@@ -3,5 +3,7 @@
 CONSOLE_ENABLE = no
 COMMAND_ENABLE = no
 MOUSEKEY_ENABLE = no
+RGBLIGHT_ENABLE = yes
```

```suggestion
RGBLIGHT_ENABLE = no
```

You don't need enable both lighting system, just RGB matrix. You can update `oryx/keymap.c:94` by removing `g_suspend_state`:
```c
if (keyboard_config.disable_layer_led) {
```

---

done